__tests__/server.test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const supertest = require('supertest');
+const { server } = require('../src/server');
+const mockRequest = supertest(server);
+const { dbInstance } = require('../src/models/index');
+const { DESCRIBE } = require('sequelize/types/query-types');
+const { INET } = require('sequelize');
+
+DESCRIBE('web server', () => {
+  beforeAll(async () => {
+    await dbInstance.sync();
+  });
+  afterAll(async () => {
+    await dbInstance.drop();
+  });
+  it('should respond with a 404 on an invalid route', () => {
+    return mockRequest.get('/foobar').then((results) => {
+      expect(results.status).toBe(404);
+    });
+  });
+  it('should respond with a 404 on an invalid method', async () => {
+    const response = await mockRequest.put('/foobar');
+    expect(response.status).toBe(404);
+  });
+  it('can create a record', async () => {
+    const person = {
+      firstName: 'MJ',
+      lastName: 'McBarksalot',
+    };
+    const response = await mockRequest.post('/people').send(person);
```

Comment:
You didn't create a people route, so this and the rest of these tests are obviously not going to work. Test the routes that you created.